### PR TITLE
fix yaml indenting of example qna

### PR DESCRIPTION
**Pull request by paulczar**
the example yaml in contributing knowledge has invalid indenting

```
31:4: [warning] wrong indentation: expected 2 but found 3 (indentation
```

### CODE REVIEW

**Comment by jjasghar** [comment]
@kelbrown20 can you verify this please?

File: README.md [hunk 1]
```diff
@@ -287,32 +287,32 @@ task_description: 'Teach the model the results of the 2024 Oscars'
...
+      Yes Billie Eilish performed "What Was I Made For?" from Barbie at the 2024 Oscars.
 document:
- repo: https://github.com/juliadenham/oscars2024_knowledge.git
- commit: e1744af
- patterns:
-   - oscars2024_results.md
+  repo: https://github.com/juliadenham/oscars2024_knowledge.git
+  commit: e1744af
+  patterns:
+    - oscars2024_results.md
```
Comment (kelbrown20) [inline README.md:315]: Based on that error your provided, just this line needs to be fixed in the example. The rest of it should be correct, unless there are other formating errors you received

Comment (paulczar) [reply README.md:315]: `The recommended indentation for YAML files is two spaces per level`   there's a bunch of single (or mixed double and single) space indents that confuses both the human and the computer trying to parse the yaml.

Given the strictness of the ilab diff (not allowing trailing spaces etc) it should also be strict about good yaml hygene and not allow single spaces.

Comment (kelbrown20) [reply README.md:315]: Oh I see what your saying, yeah its all still consistent in the PR, so this looks good to me, @juliadenham do you agree?

**Review by aakankshaduggal — COMMENTED** [review]
Thanks, lgtm!

**Review by juliadenham — APPROVED** [review]
LGTM

**Pull request merged**